File: agenda-backend/src/main/java/com/umb/agenda/agenda_backend/dto/NoteDTO.java
package com.umb.agenda.agenda_backend.dto;

import java.time.Instant;

public record NoteDTO(
  String id, String title, String content, String color,
  boolean pinned, boolean archived, Instant reminderAt,
  Instant createdAt, Instant updatedAt, Instant deletedAt
) {}

File: front/App_ToDoList_UMB-main/app/src/main/java/com/example/todo_list/network/NoteDto.kt
package com.example.todo_list.network

data class NoteDto(
    val id: String?,
    val title: String,
    val content: String,
    val color: String = "blue",
    val pinned: Boolean = false,
    val archived: Boolean = false,
    val reminderAt: String? = null,
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val deletedAt: String? = null
)
